### ISSUE TITLE
Wrong exception when searching with invalid field

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#6259]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
Wrong exception when search with invalid field: 
for example:
1 empty:""
2 space:" "
3 field name not in pre-defined fields

Currently, it reports "<BaseException: (code=1, message=invalid expression: int64 >= 0)", while the error is not invalid expression.

**Steps/Code to reproduce behavior**
Code:
>>> from pymilvus_orm import CollectionSchema, FieldSchema
>>> from pymilvus_orm import Collection
>>> from pymilvus_orm import connections
>>> from pymilvus_orm.types import DataType
>>> int64_field = FieldSchema(name="int64", dtype=DataType.INT64, is_primary=True)
>>> float_field = FieldSchema(name="float", dtype=DataType.FLOAT)
>>> float_vector = FieldSchema(name="float_vector", dtype=DataType.FLOAT_VECTOR, dim=128)
>>> schema = CollectionSchema(fields=[int64_field, float_field, float_vector])
>>> connections.connect()
>>> import numpy as np
>>> collection = Collection("test_search_collection_binbin_6", schema=schema)
>>> dim = 128
>>> nb = 500
>>> nq = 2
>>> limit = 1000
>>> import random
>>> default_search_params = {"metric_type": "L2", "params": {"nprobe": 10}}
>>> vectors = [[random.random() for _ in range(dim)] for _ in range(nb)]
>>> res = collection.insert([[i for i in range(nb)], [np.float32(i) for i in range(nb)], vectors])
>>> collection.num_entities
>>> collection.load()
>>> res = collection.search(vectors[:nq], "float_vector", default_search_params, limit, "int64 >= 0")
>>> res = collection.search(vectors[:nq], " ", default_search_params, limit, "int64 >= 0")

Results:
>>> res = collection.search(vectors[:nq], " ", default_search_params, limit, "int64 >= 0")
Error: <BaseException: (code=1, message=invalid expression: int64 >= 0)>
Error: <BaseException: (code=1, message=invalid expression: int64 >= 0)>
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/pymilvus_orm/collection.py", line 604, in search
    partition_names, output_fields, timeout, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 51, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 40, in inner
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/stub.py", line 1043, in search_with_expression
    return handler.search_with_expression(collection_name, data, anns_field, param, limit, expression, partition_names, output_fields, timeout, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 55, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 92, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 622, in search_with_expression
    return self._execute_search_requests(requests, timeout, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 63, in handler
    raise e
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 55, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 591, in _execute_search_requests
    raise pre_err
  File "/usr/local/lib/python3.6/site-packages/pymilvus/client/grpc_handler.py", line 582, in _execute_search_requests
    raise BaseException(response.status.error_code, response.status.reason)
pymilvus.client.exceptions.BaseException: <BaseException: (code=1, message=invalid expression: int64 >= 0)>

**Expected behavior**
Should not use invalid expression check message

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)
milvus: master
Name: pymilvus
Version: 2.0.0rc2.dev4
Name: pymilvus-orm
Version: 2.0.0rc2.dev15

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by czs007** [comment]
any update on this ? @FluorineDog

**Comment by ThreadDao** [comment]
The following example has the same problem. The message `Invalid expression` is so concise that it is hard to find the problem for users. 
1. `test_query_expr_not_existed_field`
```
error = {ct.err_code: 1, ct.err_msg: "fieldName(field) not found"}
```
2. `test_query_expr_unsupported_field`
```
error = {ct.err_code: 1, ct.err_msg: "column is not int64"}
```
3. `test_query_expr_non_primary_field`
```
error = {ct.err_code: 1, ct.err_msg: "column is not primary key"}
```
4. `test_query_expr_non_array_term`
```
error = {ct.err_code: 1, ct.err_msg: "right operand of the InExpr must be array"}
```
5. `test_query_expr_inconstant_term_array`
```
error = {ct.err_code: 1, ct.err_msg: "type mismatch"}
```
6. `test_query_expr_mix_term_array`
```
error = {ct.err_code: 1, ct.err_msg: "type mismatch"}
```
7. `test_query_expr_non_constant_array_term`
```
error = {ct.err_code: 1, ct.err_msg: "unsupported leaf node"}
```

**Comment by DragonDriver** [comment]
Please retest this. thx
/unassign
/assign @binbinlv

**Comment by binbinlv** [comment]
Verified and now it reports "Field   doesn't exist in schema". Maybe it is a little confused, field in scheme actually exists, just search here lacks of "Field" parameters. I think if it reports  sth like "Field   doesn't exist " may be better.

>>> res = collection.search(vectors[:nq], " ", default_search_params, limit, "int64 >= 0")
...
pymilvus.client.exceptions.ParamError: Field   doesn't exist in schema

**Comment by binbinlv** [comment]
But the issue @ThreadDao commented still exists. 

But it is about query issue, maybe it needs another issue to record them? Or just could fix through this issue either?

@DragonDriver

**Comment by binbinlv** [comment]
@ThreadDao query cases reported in your comments have been fixed by (#6344), you may enable them now in the test case.

Name: pymilvus
Version: 2.0.0rc2.dev11
Name: pymilvus-orm
Version: 2.0.0rc2.dev29
